src/types/txpool.rs

```diff
@@ -0,0 +1,204 @@
+use std::collections::HashMap;
+use crate::types::{Transaction};
+use serde::{Deserialize, Serialize};
+use crate::types::U64;
+
+/// Transaction Pool Content Info
+#[derive(Debug, Default, Clone, PartialEq, Deserialize, Serialize)]
+pub struct TxpoolContentInfo {
+    /// pending tx
+    pub pending: HashMap<String, HashMap<String, Transaction>>,
```

Comment:
Perhaps better to use `BTreeMap` here to have the keys deterministically ordered? With `HashMap` the results would be a bit random. Also we should be using more strict types than `String`. I believe it's transaction hash and the nonce, right?
```suggestion
    pub pending: BTreeMap<H256, BTreeMap<U256, Transaction>>,
```